### DISCUSSION TITLE
still getting a error on loading

### DISCUSSION THREAD

**Discussion by lgwjames** [lllyasviel/Fooocus#2995] [Q&A]
any help with this?

Last login: Sat May 25 07:38:23 on ttys001
You have mail.
/Users/Quad-Core/Desktop/fooocus.command ; exit;
(base) Quad-Core@iMac ~ % /Users/Quad-Core/Desktop/fooocus.command ; exit;
Update failed.
'refs/heads/HEAD'
Update succeeded.
[System ARGV] ['entry_with_update.py', '--theme', 'dark']
Python 3.11.5 (main, Sep 11 2023, 08:19:27) [Clang 14.0.6 ]
Fooocus version: 2.3.1
[Cleanup] Attempting to delete content of temp dir /var/folders/6l/xrkxgs657g7_vq50479x7lc40000gp/T/fooocus
[Cleanup] Cleanup successful

**Comment by mashb1t** [comment]
Please check if you have changes in git, check with `git status`. If you do have any, make sure to back the mentioned files up and run `git reset --hard`, then restart Fooocus.

**Comment by lgwjames** [comment]
thank you for the reply

(base) Quad-Core@iMac ~ % git status
fatal: not a git repository (or any of the parent directories): .git
(base) Quad-Core@iMac ~ % git reset --hard
fatal: not a git repository (or any of the parent directories): .git
(base) Quad-Core@iMac ~ %

**Reply by mashb1t** [reply]
you have to change the directory to your Fooocus folder first (the one with the config.txt), nor your user dir, then run the commands.

**Comment by lgwjames** [comment]
sorry i did think that after posting but was out

so

Last login: Sat May 25 20:05:55 on ttys000
You have mail.
(base) Quad-Core@iMac ~ % cd fooocus
(base) Quad-Core@iMac fooocus % git status      
HEAD detached at 2.3.1
Untracked files:
  (use "git add <file>..." to include in what will be committed)
	wildcards/3d-term.txt
	wildcards/NSFW_Poses 2.txt
	wildcards/NSFW_Poses 3.txt
	wildcards/NSFW_Poses.txt
	wildcards/NSFW_Poses_Full_Prompt.txt
	wildcards/NSFW_Poses_Only.txt
	wildcards/actors.txt
	wildcards/actress.txt
	wildcards/adj-architecture.txt
	wildcards/adj-beauty.txt
	wildcards/adj-general.txt
	wildcards/adj-horror.txt
	wildcards/alien.txt
	wildcards/angel.txt
	wildcards/artist-anime.txt
	wildcards/artist-black-white.txt
	wildcards/artist-botanical.txt
	wildcards/artist-c.txt
	wildcards/artist-cartoon.txt
	wildcards/artist-concept.txt
	wildcards/artist-csv.txt
	wildcards/artist-dig1.txt
	wildcards/artist-dig2.txt
	wildcards/artist-dig3.txt
	wildcards/artist-director.txt
	wildcards/artist-fantasy.txt
	wildcards/artist-fareast.txt
	wildcards/artist-fineart.txt
	wildcards/artist-horror.txt
	wildcards/artist-n.txt
	wildcards/artist-nudity.txt
	wildcards/artist-photographer.txt
	wildcards/artist-scifi.txt
	wildcards/artist-scribbles.txt
	wildcards/artist-special.txt
	wildcards/artist-surreal.txt
	wildcards/artist-ukioe.txt
	wildcards/artist-weird.txt
	wildcards/aspect-ratio.txt
	wildcards/background-color.txt
	wildcards/background.txt
	wildcards/bangs.txt
	wildcards/bdsm-type.txt
	wildcards/bdsm.txt
	wildcards/belt.txt
	wildcards/biome.txt
	wildcards/bird.txt
	wildcards/blonde.txt
	wildcards/body-fit.txt
	wildcards/body-heavy.txt
	wildcards/body-light.txt
	wildcards/body-poor.txt
	wildcards/body-short.txt
	wildcards/body-tall.txt
	wildcards/bodyshape.txt
	wildcards/bodyshape2.txt
	wildcards/bra.txt
	wildcards/braids.txt
	wildcards/breastsize.txt
	wildcards/building.txt
	wildcards/camera-manu.txt
	wildcards/camera.txt
	wildcards/cat.txt
	wildcards/celeb.txt
	wildcards/civilization.txt
	wildcards/class.txt
	wildcards/clothing-female.txt
	wildcards/clothing-male.txt
	wildcards/clothing.txt
	wildcards/corset.txt
	wildcards/cosmic-galaxy.txt
	wildcards/cosmic-nebula.txt
	wildcards/cosmic-star.txt
	wildcards/cosmic-term.txt
	wildcards/costume-female.txt
	wildcards/costume-male.txt
	wildcards/cumplay.txt
	wildcards/decade.txt
	wildcards/deity.txt
	wildcards/detail.txt
	wildcards/dinosaur.txt
	wildcards/dog.txt
	wildcards/dress.txt
	wildcards/earrings.txt
	wildcards/emoji-combo.txt
	wildcards/emoji.txt
	wildcards/expression.txt
	wildcards/eyecolor.txt
	wildcards/eyeliner.txt
	wildcards/f-stop.txt
	wildcards/fantasy-creature.txt
	wildcards/fantasy-setting.txt
	wildcards/fantasy.txt
	wildcards/female-adult.txt
	wildcards/female-young.txt
	wildcards/film-genre.txt
	wildcards/fish.txt
	wildcards/focal-length.txt
	wildcards/food.txt
	wildcards/forest-type.txt
	wildcards/fruit.txt
	wildcards/furniture.txt
	wildcards/game.txt
	wildcards/gem.txt
	wildcards/gen-modifier.txt
	wildcards/gender-ext.txt
	wildcards/gender.txt
	wildcards/genre.txt
	wildcards/hair-color.txt
	wildcards/hair-female-short.txt
	wildcards/hair-female.txt
	wildcards/hair-male.txt
	wildcards/hairaccessory.txt
	wildcards/hairlength.txt
	wildcards/hd.txt
	wildcards/headwear-female.txt
	wildcards/headwear-male.txt
	wildcards/high-heels.txt
	wildcards/horror.txt
	wildcards/identity.txt
	wildcards/interior.txt
	wildcards/iso-stop.txt
	wildcards/jessicaL.txt
	wildcards/landscape.txt
	wildcards/legwear.txt
	wildcards/lingerie.txt
	wildcards/lipstick-shade.txt
	wildcards/lipstick.txt
	wildcards/location.txt
	wildcards/makeup.txt
	wildcards/male-adult.txt
	wildcards/male-young.txt
	wildcards/monster.txt
	wildcards/movement.txt
	wildcards/national-park.txt
	wildcards/neckwear.txt
	wildcards/neg-weight.txt
	wildcards/neg.txt
	wildcards/noun-beauty.txt
	wildcards/noun-fantasy.txt
	wildcards/noun-general.txt
	wildcards/noun-horror.txt
	wildcards/noun-landscape.txt
	wildcards/noun-romance.txt
	wildcards/noun-scifi.txt
	wildcards/occupation.txt
	wildcards/oil-painting.txt
	wildcards/outfit-cottagecore.txt
	wildcards/outfit-goth.txt
	wildcards/outfit-preppy.txt
	wildcards/outfit-steampunk.txt
	wildcards/panties.txt
	wildcards/photo-term.txt
	wildcards/photoshoot-type.txt
	wildcards/planet.txt
	wildcards/pop-culture.txt
	wildcards/pop-location.txt
	wildcards/portrait-type.txt
	wildcards/public.txt
	wildcards/punk.txt
	wildcards/purse.txt
	wildcards/quantity.txt
	wildcards/race.txt
	wildcards/render-engine.txt
	wildcards/render.txt
	wildcards/robot.txt
	wildcards/rpg-Item.txt
	wildcards/scenario-fantasy.txt
	wildcards/scenario-romance.txt
	wildcards/scenario-scifi.txt
	wildcards/scenario.txt
	wildcards/scenario2.txt
	wildcards/scifi.txt
	wildcards/sculpture.txt
	wildcards/setting.txt
	wildcards/sex-act.txt
	wildcards/sex-position.txt
	wildcards/sex-toy.txt
	wildcards/ship.txt
	wildcards/site.txt
	wildcards/skin-color.txt
	wildcards/still-life.txt
	wildcards/style.txt
	wildcards/subject-fantasy.txt
	wildcards/subject-horror.txt
	wildcards/subject-romance.txt
	wildcards/subject-scifi.txt
	wildcards/subject.txt
	wildcards/suit-female.txt
	wildcards/suit-male.txt
	wildcards/superhero.txt
	wildcards/supermodel.txt
	wildcards/swimwear.txt
	wildcards/technique.txt
	wildcards/time.txt
	wildcards/train.txt
	wildcards/tree.txt
	wildcards/tribe.txt
	wildcards/trippy.txt
	wildcards/underwater.txt
	wildcards/water.txt
	wildcards/watercolor.txt
	wildcards/wave.txt
	wildcards/wh-site.txt
	wildcards/xhox_a_color.txt
	wildcards/xhox_a_material.txt
	wildcards/xhox_a_neckline.txt
	wildcards/xhox_a_occasion.txt
	wildcards/xhox_a_pattern.txt
	wildcards/xhox_a_size.txt
	wildcards/xhox_a_sleeve.txt
	wildcards/xhox_accessories.txt
	wildcards/xhox_age.txt
	wildcards/xhox_angle.txt
	wildcards/xhox_artist.txt
	wildcards/xhox_bangs.txt
	wildcards/xhox_beard.txt
	wildcards/xhox_bodytype.txt
	wildcards/xhox_bottomwear.txt
	wildcards/xhox_camera.txt
	wildcards/xhox_chest.txt
	wildcards/xhox_color.txt
	wildcards/xhox_eye.txt
	wildcards/xhox_faceexp.txt
	wildcards/xhox_faceshape.txt
	wildcards/xhox_gender.txt
	wildcards/xhox_gesture.txt
	wildcards/xhox_haircolor.txt
	wildcards/xhox_hairstyle.txt
	wildcards/xhox_hanfu.txt
	wildcards/xhox_job.txt
	wildcards/xhox_legs.txt
	wildcards/xhox_lighting.txt
	wildcards/xhox_otherfeatures.txt
	wildcards/xhox_pose.txt
	wildcards/xhox_preset.txt
	wildcards/xhox_quality.txt
	wildcards/xhox_region.txt
	wildcards/xhox_shoes.txt
	wildcards/xhox_shot.txt
	wildcards/xhox_socks.txt
	wildcards/xhox_suit.txt
	wildcards/xhox_topwear.txt
	wildcards/xhox_waist.txt

nothing added to commit but untracked files present (use "git add" to track)
(base) Quad-Core@iMac fooocus %

**Comment by lgwjames** [comment]
(base) Quad-Core@iMac fooocus % git reset --hard                                
HEAD is now at e2f9bcb docs: bump version number to 2.3.1, add changelog (#2616)
(base) Quad-Core@iMac fooocus % 

(base) Quad-Core@iMac fooocus % /Users/Quad-Core/Desktop/fooocus.command ; exit;
/Users/Quad-Core/Desktop/fooocus.command: line 1: cd: Fooocus: No such file or directory
Update failed.
'refs/heads/HEAD'
Update succeeded.
[System ARGV] ['entry_with_update.py', '--theme', 'dark']
Python 3.11.5 (main, Sep 11 2023, 08:19:27) [Clang 14.0.6 ]
Fooocus version: 2.3.1
[Cleanup] Attempting to delete content of temp dir /var/folders/6l/xrkxgs657g7_vq50479x7lc40000gp/T/fooocus
[Cleanup] Cleanup successful
Total VRAM 65536 MB, total RAM 65536 MB
Set vram state to: SHARED
Always offload VRAM
Device: mps
VAE dtype: torch.float32
Using sub quadratic optimization for cross attention, if you have memory or speed issues try using: --attention-split
Refiner unloaded.
IMPORTANT: You are using gradio version 3.41.2, however version 4.29.0 is available, please upgrade.
--------
Running on local URL:  http://127.0.0.1:7865
